src/types/Hashed.ts

```diff
@@ -0,0 +1,29 @@
+import * as bcryptC from 'bcrypt'
```

Comment:
Doesn't the coffee version switch between the native and pure js versions based upon which is installed/available? (eg in the browser the native version will never be available)